test/verify/machineslib.py

```diff
@@ -1263,6 +1278,12 @@ def fill(self):
                 b.click("#start-vm") # TODO: fix this, do not assume initial state of the checkbox
             # b.set_checked("#start-vm", self.start_vm)
 
+            # Can't use selectFromDropdown because options have same prefix
+            if (self.connection):
+                b.click("#connection button")
+                b.click("#connection li[data-data='{0}'] a".format(self.connection))
+                b.wait_in_text("#connection button", self.connectionText)
```

Comment:
With the current system "QEMU/KVM" string, this isn't actually precise enough, as both labels satisfy the pattern. If you rename the system one, this would fix itself. Otherwise, consider using `wait_text`? The full string should be pretty predictable, or does it contain other random stuff?

Reply:
I'll change the label names according to your suggestion.